content/docs/command-reference/exp/apply.md

```diff
@@ -0,0 +1,138 @@
+# exp apply
+
+Apply the changes from a previously run experiment into the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc exp apply [-h] [-q | -v] experiment
+
+positional arguments:
+  experiment     Experiment to be applied.
+```
+
+## Description
+
+This command will apply the changes from an experiment to the workspace. It is
+usually run when you are ready to Git commit the results of an experiment into
+the project.
```

Comment:
```suggestion
usually run when you are ready to commit the results of an experiment with
Git.
```

Reply:
And so do all experiment commands require the project to use Git? May be something to note prominently somewhere.